src/uu/false/src/false.rs

```diff
@@ -4,16 +4,41 @@
 //  *
 //  * For the full copyright and license information, please view the LICENSE
 //  * file that was distributed with this source code.
+use clap::{App, AppSettings, ErrorKind};
+use std::io::Write;
+use uucore::error::{set_exit_code, UResult};
 
-use clap::App;
-use uucore::error::UResult;
+static ABOUT: &str = "
+ Returns false, an unsuccessful exit status.
+
+ Immediately returns with the exit status `1`. When invoked with one of the recognized options it
+ will try to write the help or version text. Any IO error during this operation is diagnosed, yet
+ the program will also return `1`.
+";
 
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
-    uu_app().get_matches_from(args);
-    Err(1.into())
+    let app = uu_app();
+
+    // Mirror GNU options. This avoids allocation some error information which may result in yet
+    // other types of failure. See `true` for some additional comments.
```

Comment:
Even seeing them side by side, the connection to the comments in `true` is not super easy to see. Could you change this to a more self-contained comment?